lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -101,6 +104,8 @@ def trigger
       puppet_trigger['start_date'] = self.class.normalized_date("#{trigger['start_year']}-#{trigger['start_month']}-#{trigger['start_day']}")
       puppet_trigger['start_time'] = self.class.normalized_time("#{trigger['start_hour']}:#{trigger['start_minute']}")
       puppet_trigger['enabled']    = trigger['flags'] & Win32::TaskScheduler::TASK_TRIGGER_FLAG_DISABLED == 0
+      puppet_trigger['minutes_interval'] = trigger['minutes_interval'] ||= 0
+      puppet_trigger['minutes_duration'] = trigger['minutes_duration'] ||= 0
```

Comment:
I guess the guy who originally wrote the gem was inspired from the TASK_TRIGGER structure name https://msdn.microsoft.com/en-us/library/windows/desktop/aa383618(v=vs.85).aspx and I thought it would be ok to keep it like that.